app/views/presentations/_form.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= form_for @presentation do |f| %>
+  <%= render 'errors' %>
+
+  <p>
+    <%= f.label :topic %><br>
+    <%= f.text_field :topic %>
+  </p>
+
+  <p>
+    <%= f.label :description %><br>
+    <%= f.text_field :description %>
+  </p>
+
+  <p>
+    <%= f.label :duration %><br>
+    <%= f.text_field :duration %> Seconds
+  </p>
+
+  <p>
+    <%= f.label :event %><br>
+    <%= f.select :event_id, Event.all.collect { |e| [ e.name, e.id ] }, { :include_blank => '-- select event --' } %>
```

Comment:
Instead of using collect here, this would be a good situation to use the AR `:pluck` method. 
You might also want to consider using the `collection_select` rails helper

also `:include_blank => '-- select event --'` is old hash syntax